### PR TITLE
44120: Move many externals

### PR DESCRIPTION
**Pull request by nina-j**
This MR adds the POST requests necessary to move many externals between organisation units in GIR from EGIR.

Since no date fields were provided in the webform, it is assumed that validity should be from the date the request is made. This can of course be changed if necessary.

Note that this code relies on #18 to be merged first.

### CODE REVIEW

File: src/Plugin/WebformHandler/OrgunitWebformHandler.php [hunk 1]
```diff
@@ -53,10 +53,10 @@ public function alterForm(
 
       if ($externals) {
         $external_options = [];
-        foreach ($externals as $username => $e) {
-          // Get ID from username.
-          $user = user_load_by_name($username);
-          $external_options[$user->id()] = $username;
+        foreach ($externals as $external) {
```
Comment (agger-magenta) [inline src/Plugin/WebformHandler/OrgunitWebformHandler.php:56]: Why this change?

**Review by agger-magenta — APPROVED** [review]
Looks fine apart from the single comment!

File: src/Plugin/WebformHandler/OrgunitWebformHandler.php (hunk 1 above)
Comment (nina-j) [reply src/Plugin/WebformHandler/OrgunitWebformHandler.php:56]: Becuase having the initials as the user key for engagements/engagement associations is specific business logic in our test data and it will not necessarily always be the case. It's safer to get users by their GIR UUID because it'll always work regardless of the user key used.

Comment (agger-magenta) [reply src/Plugin/WebformHandler/OrgunitWebformHandler.php:56]: OK, makes sense.
